### ISSUE TITLE
Introduces severe stuttering on Steam Deck

### ISSUE THREAD

**Issue by FrostyLinky** [rebtd7/FF13Fix#93]
Game appears to run as expected out of the box on the latest version of Proton. Locked 30fps during normal gameplay with poor frame timing.
ff13fix which unlocked the frame rate but introduced severe frame drops and stuttering. Game will frequently drop to low 10s or hang in the following situations:

Loading screens and tutorials
Battle transitions prior to combat ui appearing
General exploration at certain points; couldn't find any commonalities between when these occured.

Game runs the same in default Proton, Proton GE, and Proton Experimental. 
Also tried renaming the d3d9.dll file to dxvk.dll but this appeared to result in ff13fix being disabled entirely.

**Comment by Sure-Yaa** [comment]
![download (1)](https://user-images.githubusercontent.com/43573603/184430339-7d047cac-bfa4-48a2-89ff-d227c4d2450d.jpeg)

Also, the first-line items for weapons and other item menus are cut off after applying FF13Fix on Steam Deck

**Comment by Surihix** [comment]
I don't think this fix will work properly with steam deck's default linux operating system as the fix was probably made for windows based operating system.
Your best bet would be to dual boot your steam deck with windows 10 OS and try playing this game with this fix on windows.

**Comment by NerosTie** [comment]
Same issue but on a desktop Linux + Wayland + AMD RX 6600XT.

**Comment by XarceChronus** [comment]
Same here but I can't even tell if it's working because there's no vibration

**Comment by Mupli** [comment]
any updates on this ?

**Comment by Nucleoprotein** [comment]
I dont have Steam Deck and i think @rebtd7 also so we cant test or reproduce issues, we can only guess...
I think for now this should be marked as "Not Supported"
